### PR TITLE
Get rid of common_name from tls-vars

### PR DESCRIPTION
**Pull request by joshzarrabi**
It is outdated and is not as flexible. SAN should be enough.

### CODE REVIEW

**Comment by joshzarrabi** [comment]
~@zmb3 Is there a reason to keep common names?~ This RFC says they are deprecated. https://tools.ietf.org/html/rfc6125.html#appendix-B.2. Also it is causing problems for deploying concourse with bbl on aws, as the load balancer address that aws gives you is longer than 64 characters which is too long for a common name but works for a SAN.

**Comment by zmb3** [comment]
No, not at all!  I meant to +1 this (whoops!)

I wasn't aware of the 64 character limit, just thought we eliminating some redundancy here.  I've never thought to put the long AWS name in the cert (typically create a CNAME record), but that's even more reason to do this.

File: cluster/operations/tls-vars.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@
     type: certificate
     options:
       ca: atc_ca
-      common_name: ((external_host))
       alternative_names: [((external_host))]
+      organization: atcOrg
```
Comment (vito) [inline cluster/operations/tls-vars.yml:18]: Is this related/necessary?

Comment (joshzarrabi) [reply cluster/operations/tls-vars.yml:18]: You need an organization or common name for credhub to generate the variable I think. I don't really remember why I added that haha, I'll check it out and get back to you.

**Comment by vito** [comment]
thanks!
